psalm.xml.dist

```diff
@@ -16,8 +18,10 @@
             <file name="Command/ImportMappingDoctrineCommand.php"/>
             <file name="Command/Proxy/OrmProxyCommand.php"/>
             <file name="DependencyInjection/Compiler/WellKnownSchemaFilterPass.php"/>
-            <!-- We need Psalm 5 to analyze this file -->
+            <!-- We need to run Psalm under >= PHP 8.1 to analyze this file due to intersection types (ParseError) -->
```

Comment:
Let's do that? We usually run Psalm on the latest stable PHP version on other repositories because of problems like this.

Reply:
If we do that, we might not notice when new code written cannot actually work under PHP 7.4 unless we have a test for it, this is why choice was made to run under lowest common denominator. Let's keep it for now, until list of such files keep growing

Reply:
A static analyzer is not a linter. What we do here is we run the static analysis on a subset of our classes and against outdated dependencies. We did this on the ORM and DBAL repositories and it turned out to be a bad idea.

Reply:
Alright, let's bump the version there